webpack.config.babel.js

```diff
@@ -124,12 +131,19 @@ const formatDefineHash = defineHash =>
     value => JSON.stringify(value)
   )
 
-const basePlugins = [
+const basePlugins = (bundle_name) => ([
+  new BundleAnalyzerPlugin({
+    analyzerMode: 'static',
+    reportFilename: `${__dirname}/dist/reports/bundle_${bundle_name}_size.html`,
+    defaultSizes: 'parsed'
```

Comment:
I think we should add this line `openAnalyzer: false` to prevent the report from popping out in the browser.

Reply:
Done!